docs/rfcs/0014-dsl-driven-rda-code-generation.md

```diff
@@ -315,7 +315,6 @@ The code generation plugin has a goal for each type of code to be generated.
 
 - The `entities` goal generates Hibernate database entity classes with all appropriate annotations, getters, setters, equals/hashCode, and entity relationship mappings.
 - The `transformers` goal generates data transformation/validation classes to copy data from RDA API protbuf messages into database entity classes.
-- The `csv-writers` goal generates RIF CsvWriter classes for each parent entity.
 - The `sql` goal generates sample SQL DDL for each table that can be used as the basis for creating Flyway migration files.
```

Comment:
I don't think we need to alter RFCs.  They are more a point in time about a particular discussion.